integration-tests/data/results/cca.freshness-fail.json

```diff
@@ -1,18 +1,20 @@
-{
-        "ear.appraisal-policy-id": "policy:CCA_SSD_PLATFORM",
-        "ear.status": "contraindicated",
-        "ear.trustworthiness-vector": {
-            "configuration": 99,
-            "executables": 99,
-            "file-system": 99,
-            "hardware": 99,
-            "instance-identity": 99,
-            "runtime-opaque": 99,
-            "sourced-data": 99,
-            "storage-opaque": 99
-        },
-        "ear.veraison.policy-claims": {
-            "problem": "integrity validation failed"
-        }
-}
+{ 
+    "ARM_CCA" :  {
+            "ear.appraisal-policy-id": "policy:ARM_CCA",
+            "ear.status": "contraindicated",
+            "ear.trustworthiness-vector": {
+                "configuration": 99,
+                "executables": 99,
+                "file-system": 99,
+                "hardware": 99,
+                "instance-identity": 99,
+                "runtime-opaque": 99,
+                "sourced-data": 99,
+                "storage-opaque": 99
+            },
+            "ear.veraison.policy-claims": {
+                "problem": "integrity validation failed"
+            }
+    }
 
+}
```

Comment:
Indentation has gone a bit wild :-)

Can you please do a `jq` sweep over the JSON files you have touched?

Something like:
```sh
git diff --name-only main \
  | grep "\.json$" \
  | while read f ; do [ -f $f ] && ( jq . $f | sponge $f ) ; done
```
should do the trick.

(Note that to run the above you may need to install `sponge(1)`.  On Debian/Ubuntu, do `sudo apt-get install moreutils`.)

Reply:
PS: also worth going through the READMEs that you have touched to make sure that any JSON example is properly formatted.

Reply:
Thanks, Totally agree with you, I will run `jq` with all impacted `json` files and update the same!

Reply:
Done all the JSON files proper indentation!